src/modules/math.wren

```diff
@@ -135,27 +135,21 @@ var NumToHex = Fn.new {|num|
   return strings.join("")
 }
 
-var HexToNum = Fn.new {|hex|
-  var first = hex[0]
-  var second = hex[1]
-  if (48 <= first && first <= 57) {
-    first = first - 48
-  } else if (65 <= first && first <= 70) {
-    first = 10 + first - 65
-  } else if (97 <= first && first <= 102) {
-    first = 10 + first - 97
-  } else {
-    Fiber.abort("Invalid hex")
-  }
-  if (48 <= second && second <= 57) {
-    second = second - 48
-  } else if (65 <= second && second <= 70) {
-    second = 10 + second - 65
-  } else if (97 <= second && second <= 102) {
-    second = 10 + second - 97
+var ToNum = Fn.new {|digit|
```

Comment:
We might want to rename this "CharToNum" or "ByteToNum" just to be clear what its purpose is.

Reply:
I followed the convention of `NumToHex` -> `ToHex`, hence `HexToNum` -> `ToNum`.

Reply:
But `CharToNum` really seems reasonable.
I'll change that to `HexDigitToNum`.

Reply:
I missed that connection, apologies. 

We might as well rename the `toHex` as well, to be `NumToHexDigit` or similar, for symmetrical clarity.

Reply:
Yes but I don't want to touch the existing code in this PR

Reply:
If you want I can do that of course, though